packages/help-center/src/components/help-center-gpt.tsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-restricted-imports */
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+import { useJetpackSearchAIQuery } from '@automattic/data-stores';
+import { external, Icon, page } from '@wordpress/icons';
+import { useI18n } from '@wordpress/react-i18n';
+import { useEffect } from 'react';
+import { stripHTML } from 'calypso/lib/formatting';
+export const SITE_STORE = 'automattic/site' as const;
+import './help-center-article-content.scss';
+
+type Props = {
+	message: string;
+	setLoadingState: CallableFunction;
+};
+
+export function HelpCenterGPT( { message = '', setLoadingState }: Props ) {
+	const { __ } = useI18n();
+
+	const { data, isFetching } = useJetpackSearchAIQuery( '9619154', message, 'response' );
+	// Report loading state up.
+	useEffect( () => {
+		if ( setLoadingState ) {
+			setLoadingState( isFetching );
+		}
+	}, [ isFetching, setLoadingState, message ] );
+
+	return (
+		<div className="help-center-sibyl-articles__container">
+			<h3 id="help-center--contextual_help" className="help-center__section-title">
+				{ __( 'Quick response', __i18n_text_domain__ ) }
+			</h3>
+			<div className="help-center-gpt-response">
+				{ data?.response && stripHTML( data.response ) }
```

Comment:
On the backend we start with Markdown, and transform to HTML just before returning the response.

The main options I see for displaying the actual formatting:
- use `dangerouslySetInnerHTML` - I think it's reasonably safe here but dangerous if the backend changes enough to introduce the risk of malicious HTML. 
- Return markdown and use something like  [react-markdown](https://github.com/remarkjs/react-markdown) to render it.

@artpi Any preference?

Reply:
I was searching for a component similar to `kses` that would do `dangerouslySetInnerHTML` but only permit certain tags/attrs - it sounds like there should be something like this.
I am leaning towards HTML but I can also introduce a parameter for the endpoint to return markdown.

Reply:
You can use [`stripTags`](https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/formatting/strip-html.js#L10). It allows you to whitelist tags.